README.md

```diff
@@ -15,6 +15,7 @@ SciKeras attempts to resolve these issues by providing maintained, documented wr
 entire Scikit-Learn and Keras ecosystems. Some advantages over the TF wrappers are:
```

Comment:
I still suggest this outline for README.md:

> **Why should I use SciKeras?**
> The motivation is at https://scikeras.readthedocs.io/
>
> **How do I transition from TF?**
> See https://scikeras.readthedocs.io/en/latest/quickstart.html
>
> **Basic development workflow**
> 1. ...
> 2. ...

I think the only people who read READMEs are people browsing GitHub (so links don't bother them), and developers (who aren't interested in these questions anyways).

Reply:
I commented README.md because it showed up first in GitHub's diff, but it looks like my comments also apply to index.rst.

Reply:
Got it. I'll mirror changes in README.md to index.rst (or a new page, as discussed).